src/OrchardCore/OrchardCore.DisplayManagement.Liquid/TagHelpers/LiquidTagHelperActivator.cs

```diff
@@ -52,15 +52,15 @@ public LiquidTagHelperActivator(Type type)
                 }
 
                 var setter = MakeFastPropertySetter(type, property);
+                var viewContextAttribute = property.GetCustomAttribute<ViewContextAttribute>();
+                if (viewContextAttribute != null && property.PropertyType == typeof(ViewContext))
+                {
+                    _viewContextSetter = (helper, context) => setter(helper, context);
```

Comment:
just assign property name to new variable
`var viewContentProperty`

Reply:
Yes, regarding the existing code @ns8482e is right.

But because viewContext can't be html bound => Looks good to me

Maybe a new line after line 54 and / or line 55

I would have put line 54 to 60 at the beginning of the loop

Reply:
> I would have put line 54 to 60 at the beginning of the loop

Why? I think it will be executed more than what expected, unless there's a proper reason, I will change the code for what you are suggesting, but hopefully you answer my questions, so everything will be clear for me ;)